sharks/src/share_ff.rs

```diff
@@ -14,7 +14,7 @@ pub const FIELD_ELEMENT_LEN: usize = 24;
 #[cfg_attr(feature = "fuzzing", derive(Arbitrary))]
 #[cfg_attr(feature = "zeroize_memory", derive(Zeroize))]
 #[derive(PrimeField)]
-#[PrimeFieldModulus = "680564733841876926926749214863536421547"] // 2^129 - 1365
+#[PrimeFieldModulus = "340282366920938463463374607431768223907"] // 2^128 + 12451 (https://eprint.iacr.org/2011/326)
```

Comment:
Please move the comment to its own line above, to keep the line length shorter and easier to read.